addons/test_mail/tests/test_performance.py

```diff
@@ -281,7 +281,7 @@ def test_mail_composer_w_template(self):
         test_record = self.env['mail.test.ticket'].browse(self.test_record_full.id)
         test_template = self.env['mail.template'].browse(self.test_template_full.id)
         # TODO FIXME non deterministic, last check 25 Mar 2020. Runbot at most +7 compared to local.
-        with self.assertQueryCount(__system__=12, emp=14):
+        with self.assertQueryCount(__system__=13, emp=15):
```

Comment:
This might be problematic but I will let that to our favorite reviewer.

Reply:
I think this comes from new fields of mail.test.ticket. I would instead use another model to test tracking details and keep ticket model as it is to keep coherency through versions.